packages/component-library/src/components/form/mt-text-editor/_internal/mt-text-editor-toolbar-button.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <button
+    @click="handleButtonClick()"
+    :class="buttonClass(button)"
+    v-tooltip="{
+      disabled: !translatedLabel,
+      message: translatedLabel,
+      position: 'top',
+      hideDelay: 0,
+    }"
+    :aria-label="translatedLabel"
+    :disabled="button.disabled ? button.disabled(props.editor) : disabled"
+  >
+    <mt-icon v-if="button.icon" :name="button.icon" />
+    <span v-else>{{ translatedLabel }}</span>
+  </button>
+</template>
+
+<script setup lang="ts">
+import { computed, type PropType } from "vue";
+import type { CustomButton } from "./mt-text-editor-toolbar.vue";
+import type { Editor } from "@tiptap/vue-3";
+import mtIcon from "@/components/icons-media/mt-icon/mt-icon.vue";
+import vTooltip from "@/directives/tooltip.directive";
+import { useI18n } from "@/composables/useI18n";
+
+const { t, te } = useI18n({
+  messages: {},
+});
+
+const props = defineProps({
+  editor: {
+    type: Object as PropType<Editor>,
+    required: true,
+  },
+  button: {
+    type: Object as PropType<CustomButton>,
+    required: true,
+  },
+  disabled: {
+    type: Boolean,
+    default: false,
+  },
+});
+
+const emit = defineEmits(["click"]);
+
+const handleButtonClick = () => {
+  emit("click");
+};
+
+const buttonClass = (button: CustomButton) => {
+  return {
+    "mt-text-editor-toolbar-button": true,
+    "is-active": button.isActive && button.isActive(props.editor),
+  };
+};
+
+const translatedLabel = computed(() => {
+  if (te(props.button.label)) {
+    return t(props.button.label);
+  }
+
+  return props.button.label;
```

Comment:
I don't think the `te` method is needed. If it can't resolve `path.to.translation` it returns the literal path to the token. See this [test](https://github.com/shopware/meteor/blob/main/packages/component-library/src/composables/useI18n.spec.ts#L20)